components/count-badge/count-badge.js

```diff
@@ -134,7 +173,15 @@ class CountBadge extends RtlMixin(LitElement) {
 			numberString = `${'9'.repeat(this.maxDigits)}+`;
 		}
 		return html`
-        	<div class="d2l-count-badge-number">${numberString}</div>`;
+        	<div
+			class="d2l-count-badge-number" 
+			id="${this._badgeId}"
+			tabindex="${ifDefined(this.tabStop || this.hasTooltip ? '0' : undefined)}" 
+			role="${ifDefined(this.announceChanges ? 'status' : undefined)}">
+					<div aria-hidden="true">${numberString}</div>
```

Comment:
I decided to get rid of the aria-label altogether, and choose either a tooltip or d2l-offscreen as the accessible label.

Reply:
Cool. In that case though you'll need to manually link the `span` and the badge together (tooltip does this for you) by giving the `span` a generated `id` and then putting `aria-labelledby` on the badge.

Reply:
Hmm, actually since it's a child that may not be necessary actually. But then I'm thinking maybe the tooltip shouldn't be a child since it will create that linkage for you.

Reply:
It seems to be working without the aria-labelledby and the screenreader gets the text in either case. I'm also applying an aria-describedby to the filter menu items and they can get the text from the d2l-offscreen as well... so I didn't need the aria-label in either case.